### PR TITLE
Add repos to FI working group, reflects incubator transfers

### PR DESCRIPTION
**Pull request by christopherclark**

### CODE REVIEW

**Review by rkoster — APPROVED** [review]
LGTM

**Review by beyhan — APPROVED** [review]

**Review by loewenstein — APPROVED** [review]

**Comment by christopherclark** [comment]
Accidentally edited PR branch while this was outstanding.  Please disregard those last three commits - they negate themselves - will include in separate PR.

**Review by stephanme — APPROVED** [review]

**Review by LeePorte — APPROVED** [review]

**Review by emalm — APPROVED** [review]

File: toc/working-groups/foundational-infrastructure.md [hunk 1]
```diff
@@ -62,50 +62,59 @@ areas:
...
+  - cloudfoundry/switchboard
 - name: System Logging and Metrics (rsyslog / event-log)
   approvers:
   - name: Ben Fuller
     github: Benjamintf1
   repositories:
-  - cloudfoundry-incubator/event-log-release
+  - cloudfoundry/blackbox
+  - cloudfoundry/event-log-release
+  - cloudfoundry/metrics-release
```
Comment (rkoster) [inline toc/working-groups/foundational-infrastructure.md:117]: @christopherclark this repo does not seem to exist: https://github.com/cloudfoundry/metrics-release

This is causing github projects sync to fail: https://github.com/cloudfoundry/community/runs/6387544577?check_suite_focus=true

Comment (christopherclark) [reply toc/working-groups/foundational-infrastructure.md:117]: Ah! I've put in a new [PR (272)](https://github.com/cloudfoundry/community/pull/272), which corrects this.

**Pull request merged**